t/cl-emoji.lisp

```diff
@@ -11,13 +11,18 @@
 		    :cl-emoji (pathname (format nil "data/emoji_~a.lisp"
                                         cl-emoji::*current-version*))))
   (let ((emoji-list (read s)))
-    (plan (+ 5 (length emoji-list)))
+    (plan (+ 6 (length emoji-list)))
     (dolist (u emoji-list)
       (is (length u) 6))
     (is "😀" (emoji:codepoint '("U+1F600")))
     (is "😁" (emoji:name "grinning face with smiling eyes"))
     (ok (< 0 (length (emoji:annotation "blue"))))
     (ok (< 0 (length (emoji:group "Smileys & People"))))
-    (ok (< 0 (length (emoji:subgroup "clothing"))))))
+    (ok (< 0 (length (emoji:subgroup "clothing"))))
+    (with-emoji-list (emoji-list)
```

Comment:
Why did you call `with-emoji-list` here?
The macro's body has no `emoji-list` as the local variable.